source/fundamentals/builders/aggregates.txt

```diff
@@ -11,3 +11,506 @@ Aggregates Builders
    :class: singlecol
 
 .. _aggregates-builders:
+
+This guide provides an overview of the :java-core-api:`Aggregates </com/mongodb/client/model/Aggregates>`
+class which provides static factory methods that build :manual:`aggregation pipeline
+operators </reference/operator/aggregation/>`.
+
+For a more thorough introduction to Aggregation, see our :ref:`Aggregation guide <java-aggregation>`.
+
+.. tip::
+
+   For brevity, you may choose to import the methods of the ``Aggregates`` class
```

Comment:
This is importing more than just Aggregates class methods.  Can you update the text to match the actual imports?